content/kotlin/concepts/classes/classes.md

```diff
@@ -0,0 +1,117 @@
+---
+Title: 'Classes'
+Description: 'A class definition in Kotlin consists of the class keyword, followed by the name of the class, and a block of code enclosing the properties and functions of the class.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Classes'
+  - 'Objects'
+  - 'Data Types'
+  - 'Constructors'
+  - 'Methods'
+  - 'OOP'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+In Kotlin **classes** are a blueprint for creating objects. They can contain properties (data) and functions (behavior).
+
+## Syntax
+
+```pseudo
+fun main(){
+    class ClassName {
+    // properties
+    // functions
+    }
+}
+```
+
+Here are some rules for defining classes in Kotlin:
+
+- The name of a class should start with a capital letter and should be a noun (e.g. Dog, Person, BankAccount).
+- A class can have properties (data) and functions (behavior).
+- Properties and functions of a class can be made public (accessible from anywhere) or private (accessible only within the class). By default, they are public in Kotlin.
+- A class can have a primary constructor and/or one or more secondary constructors. The primary constructor is defined in the class header, and the secondary constructors are defined using the `constructor` keyword.
+- A class can have an `open`, `abstract`, `sealed`, or `final` modifier. The `open` modifier allows the class to be subclassed, the `abstract` modifier allows the class to have abstract (unimplemented) functions, the `sealed` modifier allows the class to have subclasses, but only in the same file, and the `final` modifier prevents the class from being subclassed.
```

Comment:
Is it possible to label where the modifier goes in the syntax? Perhaps label the position in the syntax as `<modifier>`. And mention here that modifiers are optional.

Reply:
Agree with mentioning where it goes in the syntax. While I don't see a problem with the modifiers being here, I agree that there should be a Kotlin entry on inheritance.

Reply:
Ok, we'll leave the modifiers here and just add where the modifiers would be written in the syntax. 

Also, I'll open up an issue for a Kotlin entry on inheritance. 

cc @dakshdeepHERE